src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -3385,4 +3388,69 @@ public void testWrapIfMissing_StringString() {
         assertSame("ab/ab", StringUtils.wrapIfMissing("ab/ab", "ab"));
         assertSame("//x//", StringUtils.wrapIfMissing("//x//", "//"));
     }
+    @Test
+    void testReverse() {
+        assertEquals("cba", StringUtils.reverse("abc"));
+        assertNull(StringUtils.reverse(null));
+    }
+
+    @Test
+    void testCountCharacters() {
+        Map<Character, Integer> expected = new HashMap<>();
+        expected.put('a', 2);
+        expected.put('b', 1);
+        assertEquals(expected, StringUtils.countCharacters("aba"));
+        assertNull(StringUtils.countCharacters(null));
+    }
+
+    @Test
+    void testIsPalindrome() {
+        assertTrue(StringUtils.isPalindrome("madam"));
+        assertFalse(StringUtils.isPalindrome("hello"));
+        assertFalse(StringUtils.isPalindrome(null));
+    }
+
+    @Test
+    void testToTitleCase() {
+        assertEquals("Hello World", StringUtils.toTitleCase("hello world"));
+        assertNull(StringUtils.toTitleCase(null));
+    }
+
+    @Test
+    void testLongestCommonPrefix() {
+        assertEquals("abc", StringUtils.longestCommonPrefix("abcdef", "abcxyz"));
+        assertEquals("", StringUtils.longestCommonPrefix("abc", "xyz"));
+        assertNull(StringUtils.longestCommonPrefix(null, "abc"));
+    }
+
+    @Test
+    void testRepeat() {
+        assertEquals("abcabcabc", StringUtils.repeat("abc", 3));
+        assertNull(StringUtils.repeat(null, 3));
+    }
+
+    @Test
+    void testIsNumeric() {
+        assertTrue(StringUtils.isNumeric("12345"));
+        assertFalse(StringUtils.isNumeric("123a45"));
+        assertFalse(StringUtils.isNumeric(null));
+    }
+
+    @Test
+    void testCapitalizeWords() {
+        assertEquals("Hello World", StringUtils.capitalizeWords("hello world"));
```

Comment:
-1: General note: There is no difference between this testing of `capitalizeWords` and `toTitleCase` so how do you know the methods are doing the proper work?